apps/web/src/components/app/player/PlayerActions.test.tsx

```
@@ -0,0 +1,185 @@
+}));
+
+vi.mock('@/components/ui/dropdown-menu', () => ({
+  DropdownMenu: ({ children }: PropsWithChildren) => <div>{children}</div>,
+  DropdownMenuContent: ({ children }: PropsWithChildren) => <div>{children}</div>,
+  DropdownMenuSeparator: () => <hr />,
+  DropdownMenuTrigger: ({ children }: PropsWithChildren) => <>{children}</>,
+  DropdownMenuCheckboxItem: ({
+    children,
+    onCheckedChange,
+  }: PropsWithChildren<{ onCheckedChange?: (v: boolean) => void }>) => (
+    <button role="menuitemcheckbox" onClick={() => onCheckedChange?.(true)}>
```

Biome (2.4.4)

[error] The element with the menuitemcheckbox ARIA role does not have the required ARIA attributes.

(lint/a11y/useAriaPropsForRole)